client/src/components/Post/Post.tsx

```diff
@@ -248,12 +299,46 @@ class Post extends Component<Props, State> {
                     <div className={'post-comments-container'}>
                         {
                             this.state.comments.map((comment, index) => (
-                                <div key={index} className={'comment-card'}>
+                                <div key={index} className={'comment-card'}
+                                     onMouseEnter={() => this.setIsShown(index)}
+                                     onMouseLeave={() => this.setIsShown(-1)}>
                                     <div className={'comment-header'}>
                                         {comment.username + ' - ' + comment.creationDate + ' '}
-                                        {comment.editDate && <span style={{color: '#818181'}}>edited</span>}
+                                        {comment.editDate && <span style={{color: '#818181', marginLeft: 5}}>edited</span>}
+                                        {
+                                            (parseJwt(localStorage.getItem('token')).role.includes('ROLE_ADMIN') ||
+                                                parseJwt(localStorage.getItem('token')).userId == this.state.post.userId)  && this.state.hoverIndex==index &&
```

Comment:
@sofiasdz delete comment option should only appear for **admins** or **comment owners**, not post owners. Per user stories:
`Como un​ usuario autor de un comentario o un usuario administrador, quisiera​ poder eliminar un comentario, de modo que​ desaparezca de la plataforma y no se pueda interactuar más con el mismo`

Align w/ Facu (very similar correction below)